tests/results/DiffTest.py

```diff
@@ -60,31 +60,36 @@ def test_double_changes_with_same_diff(self):
             self.fail('We should not have a conflict on same diff!')
 
     def test_affected_code(self):
```

Comment:
I think this test should be split into a deprecated part and the rest so we can deprecate the old usage and then remove it?